torchaudio/functional.py

```diff
@@ -458,68 +458,67 @@ def phase_vocoder(
     factor of ``rate``.
 
     Args:
-        complex_specgrams (Tensor): Dimension of `(..., freq, time, complex=2)`
```

Comment:
How about something like this?
```
        complex_specgrams (Tensor): Either a real tensor of dimension of `(..., freq, time, complex=2)`
            or a tensor of dimension `(..., freq, time)` with complex dtype.
```
We were using "complex tensor" to mean `(..., complex=2)`. This is now ambiguous. What expression do you recommend to refer to a tensor of complex dtype? "tensor with a complex dtype"?

Reply:
yeah tensor with a complex dtype sounds good. However this "or" way of documenting could be problematic in case where the function takes more than one complex tensors. Perhaps in those cases, we can add a note. 

I think it might be nicer to add a separate example with complex dtype tensors so that it's also clear that the returned output would also be complex (if applicable) etc., especially since we are planning to switch to using complex dtype tensors in the release after the upcoming release

Reply:
I do find this "or" way of discussing this a little cumbersome, and I agree this will get long if many tensors are involved. We could add a note in each, and just define the args/returns with complex dtype. We can still keep the example for clarity.

```python
"""
    We are migrating to complex-dtype tensors. For backward compatibility reason,
    this function still supports the legacy convention of ending with a dimension of 2
    to represent a complex tensor.

    Args:
        complex_specgrams (Tensor): A tensor of dimension `(..., freq, time)` with complex dtype.
        rate (float): Speed-up factor
        phase_advance (Tensor): Expected phase advance in each bin. Dimension of (freq, 1)
    Returns:
        Tensor: Complex Specgrams Stretch with dimension of `(..., freq, ceil(time/rate))`
            with a complex dtype.

    Example

    Example - Legacy
"""
```
Thoughts?

Reply:
P.S. Good suggestion [below](https://github.com/pytorch/audio/pull/758/files#r466610938) for example naming